src/exercises/exerciseTypes/SpeakButton.js

```diff
@@ -67,44 +65,83 @@ export default function SpeakButton({ bookmarkToStudy, api, styling }) {
 
   return (
     <>
-      {isSpeaking && (
-        <s.FeedbackButton
-          disabled={true}
-          style={{ marginLeft: buttonLeftMargin }}
+      {handleClick !== undefined && (
```

Comment:
it's hard for me to see exactly what happens here, but there seems to be A LOT of duplication.
if i'm right, the only difference between the two blocks is that if `handleClick` is defined, you call it and otherwise not? 

couldn't you fix this by just adding the condition inside the onClick() from SpeakButton? e.g. something like: 

```
<s.SpeakButton
          disabled={isSpeaking}
          onClick={() => {
            !isSpeaking && handleSpeak();
            if (handleClick ~== undefined) {handleClick();}
          }}
        >
          {isSpeaking && (
            <Loader
              type="Bars"
              color="#ffffff"
              width={style.loader_width}
              height={style.loader_height}
              style={{
                paddingLeft: style.loader_paddingLeft,
                paddingRight: style.loader_paddingRight,
                marginTop: style.loader_marginTop,
                marginBottom: style.loader_marginBottom,
                display: "flex",
                justifyContent: "center",
              }}
            />
          )}

          {!isSpeaking && (
            <img
              src="/static/images/volume_up.svg"
              alt={strings.speak}
              width={style.img_width}
              height={style.img_height}
              style={{
                paddingLeft: style.img_paddingLeft,
                paddingRight: style.img_paddingRight,
              }}
            />
          )}
        </s.SpeakButton>

```